src/xcode/ENA/ENA/Source/Services/DeadmanNotification/DeadmanNotificationManager.swift

```diff
@@ -26,31 +26,33 @@ struct DeadmanNotificationManager: DeadmanNotificationManageable {
 	/// Schedules a local notification to fire 36 hours from now, if there isn´t a notification already scheduled
 	func scheduleDeadmanNotificationIfNeeded() {
 		/// Check if Deadman Notification is already scheduled
-		userNotificationCenter.getPendingNotificationRequests { notificationRequests in
-			if notificationRequests.contains(where: { $0.identifier == Self.deadmanNotificationIdentifier }) {
-				/// Deadman Notification already setup -> return
-				return
-			} else {
-				/// No Deadman Notification setup, continue to setup a new one
-				let content = UNMutableNotificationContent()
-				content.title = AppStrings.Common.deadmanAlertTitle
-				content.body = AppStrings.Common.deadmanAlertBody
-				content.sound = .default
-				
-				let trigger = UNTimeIntervalNotificationTrigger(
-					timeInterval: 36 * 60 * 60,
-					repeats: false
-				)
-				
-				let request = UNNotificationRequest(
-					identifier: Self.deadmanNotificationIdentifier,
-					content: content,
-					trigger: trigger
-				)
-				
-				userNotificationCenter.add(request) { error in
-					if error != nil {
-						Log.error("Deadman notification could not be scheduled.")
+		if !CWAHibernationProvider.shared.isHibernationState {
```

Comment:
Instead of this, I would prefer to have a check on the function itself. Simply don't call it if it is in hibernation mode.